webui/portal/src/main/java/org/exoplatform/portal/application/PortalRequestContext.java

```diff
@@ -295,8 +296,12 @@ public UserPortal getUserPortal() {
     }
 
     public UserPortalConfig getUserPortalConfig() {
+        String remoteUser = "";
```

Comment:
Should be null if not connected IMO

```suggestion
        String remoteUser = null;
```

Reply:
Ok